### PR TITLE
Added ability to save as an org

### PR DESCRIPTION
**Pull request by dane-johnson**
## Description

Fixes bug where a user cannot save to a snippet owned by an org she is a member of.

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/containers/AppBody.jsx [hunk 1]
```diff
@@ -112,8 +112,8 @@ export class AppBody extends Component {
 
         const permissions = {
           canRead: true,
-          // Currently, users may only edit a file they own
-          canEdit: (username === cookie.load('username')),
+          canEdit: (username === cookie.load('username') ||
```
Comment (thebho) [inline src/containers/AppBody.jsx]: This is kind of confusing with username meaning 2 different things.  Should it be tokenUser, tokenOrgs, and snippetUser?

Comment (thebho) [inline src/containers/AppBody.jsx]: Maybe even snippetOwner

Comment (dane-johnson) [inline src/containers/AppBody.jsx]: This code section is very old and in dire need of a refactor. @solkaz is working on that now.

Comment (solkaz) [inline src/containers/AppBody.jsx]: I already have refactored that in [this PR](https://github.com/maryvilledev/codesplainUI/pull/464)

**Comment by thebho** [comment]
Go it.  Just wanted to point it out

**Pull request merged**